### PR TITLE
#30714 - home page - container structure and banners

### PR DESCRIPTION
**Pull request by anqaka**
https://dev.snowdog.pro/issues/30714

### CODE REVIEW

File: Magento_Theme/templates/html/title.phtml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <h1 class="heading"
+        <?php if ($block->getId()): ?> id="<?php /* @escapeNotVerified */ echo $block->getId();?>" <?php endif; ?>
+        <?php if ($block->getAddBaseAttributeAria()): ?>
+            aria-labelledby="<?php /* @escapeNotVerified */ echo $block->getAddBaseAttributeAria(); ?>"
+        <?php endif; ?>>
+        <?php /* @escapeNotVerified */ echo $title ?>
+    </h1>
+    <?php echo $block->getChildHtml(); ?>
+</div>
+<?php endif; ?>
```
Comment (Igloczek) [inline Magento_Theme/templates/html/title.phtml]: Ough, this file looks terrible...

```php
<?php if (!empty($block->getPageHeading())): ?>
    <div class="container <?= /* @escapeNotVerified */ $block->getCssClass(); ?>">
        <h1 class="heading"
            <?= $block->getId() ? 'id:"' . $block->getId() . '"' : '' ?>
            <?= $block->getAddBaseAttributeAria() ? 'aria-labelledby="' . $block->getAddBaseAttributeAria() . '"' ?>
        >
            <span class="base" 
                  data-ui-id="page-title-wrapper"
                  <?= $block->getAddBaseAttribute() ?>
            >
                <?= /* @escapeNotVerified */ $block->escapeHtml($block->getPageHeading()) ?>
            </span>
        </h1>
        <?php echo $block->getChildHtml(); ?>
    </div>
<?php endif; ?>
```

File: styles/_base.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+body {
+    font-family: $font-family-sans-serif;
+    font-size: $font-size-base;
+    color: $font-color-base;
+}
+
+p {
+    margin: 0;
+    padding: 0;
+}
```
Comment (Igloczek) [inline styles/_base.scss]: Do not add it here, it's part of `globals/typography/paragraph`

File: Magento_Cms/layout/cms_index_index.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="main" htmlClass="container home">
+            <container name="home-wrapper-top"
```
Comment (Igloczek) [inline Magento_Cms/layout/cms_index_index.xml]: Let's refactor this code by moving as much as you can to `phtml` templates (including CMS blocks loading)

File: Magento_Cms/styles/module/_home.scss [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    &__column-left {
+        padding-right: 15px;
+    }
+
+    &__column-right {
+        padding-left: 15px;
+    }
+
+    &__tier-banner {
```
Comment (Igloczek) [inline Magento_Cms/styles/module/_home.scss]: What does it mean?

File: Magento_Theme/templates/html/title.phtml (hunk 1 above)
Comment (anqaka) [inline Magento_Theme/templates/html/title.phtml]: fixed

File: styles/_base.scss (hunk 2 above)
Comment (anqaka) [inline styles/_base.scss]: removed

File: Magento_Cms/layout/cms_index_index.xml (hunk 3 above)
Comment (anqaka) [inline Magento_Cms/layout/cms_index_index.xml]: moved to phtml files

File: Magento_Cms/styles/module/_home.scss (hunk 4 above)
Comment (anqaka) [inline Magento_Cms/styles/module/_home.scss]: sorry, I mixed up languages, I wanted to named it 'third' - like a third part banner. tier it's from french. corrected to `third-banner`

File: styles/modules/_banners.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
+$banner-default-position-value: 10%;
+$banner-top-value             : $banner-default-position-value;
+$banner-bottom-value          : $banner-default-position-value;
+$banner-left-value            : $banner-default-position-value;
+$banner-right-value           : $banner-default-position-value;
+
+.banner {
```
Comment (Igloczek) [inline styles/modules/_banners.scss]: Looks like 1:1 copy of component styles...

File: styles/styles.scss [hunk 6]
```diff
@@ -42,3 +42,11 @@
 // alpaca components' modules
 @import '../Snowdog_AlpacaComponents/03-modules/products-grid/products-grid';
 @import '../Snowdog_AlpacaComponents/03-modules/products-list/products-list';
+
+// theme components
+@import '../Magento_Cms/styles/module';
+
+
+// new theme components, to move to components repo
+// update existing components
```
Comment (Igloczek) [inline styles/styles.scss]: Seems like it can be removed now.

File: styles/vars/_theme-variables.scss [hunk 7]
```diff
@@ -1 +1,3 @@
 $box-shadow: -5px 5px 7px 0 rgba(0, 0, 0, 0.2);
+$button-icon-border: 0 none;
```
Comment (Igloczek) [inline styles/vars/_theme-variables.scss]: `none` will be probably enough.

File: Magento_Cms/styles/module/_home.scss [hunk 8]
```diff
@@ -0,0 +1,41 @@
+.home {
```
Comment (Igloczek) [inline Magento_Cms/styles/module/_home.scss]: Technically, this should be a `home` view component, but for now, leave it as it is.

Comment (anqaka) [inline Magento_Cms/styles/module/_home.scss]: ok

File: styles/vars/_theme-variables.scss (hunk 7 above)
Comment (anqaka) [inline styles/vars/_theme-variables.scss]: fixed

File: styles/styles.scss (hunk 6 above)
Comment (anqaka) [inline styles/styles.scss]: removed

File: styles/modules/_banners.scss (hunk 5 above)
Comment (anqaka) [inline styles/modules/_banners.scss]: 👍  I forgot to removed it after adding banners to components. removed

**Pull request merged**